pkgs/development/libraries/libsystemtap/default.nix

```diff
@@ -0,0 +1,27 @@
+{stdenv, fetchgit, gettext, python, elfutils}:
+
+stdenv.mkDerivation {
+  name = "libsystemtap";
+  src = fetchgit {
+    url = git://sourceware.org/git/systemtap.git;
+    rev = "4051c70c9318c837981384cbb23f3e9eb1bd0892";
+    sha256 = "0sd8n3j3rishks3gyqj2jyqhps7hmlfjyz8i0w8v98cczhhh04rq";
+    fetchSubmodules = false;
+  };
+
+  dontBuild = true;
+
+  nativeBuildInputs = [ gettext python elfutils ];
```

Comment:
Given `dontBuild` is set to true are all of these used?

Reply:
Yap, all three are being used in the `configure` phase. Any better place to put them?

Reply:
Ah sorry didn't catch that. Please ignore my comment, you're spot on.